README.md

```diff
@@ -369,4 +370,242 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
+--set alfresco-content-services.repository.replicaCount=1 ^
+--set alfresco-content-services.repository.livenessProbe.initialDelaySeconds=420 ^
+--set alfresco-content-services.pdfrenderer.livenessProbe.initialDelaySeconds=300 ^
+--set alfresco-content-services.libreoffice.livenessProbe.initialDelaySeconds=300 ^
+--set alfresco-content-services.imagemagick.livenessProbe.initialDelaySeconds=300 ^
+--set alfresco-content-services.share.livenessProbe.initialDelaySeconds=420 ^
+--set alfresco-content-services.repository.resources.requests.memory="2000Mi" ^
+--set alfresco-content-services.pdfrenderer.resources.requests.memory="500Mi" ^
+--set alfresco-content-services.imagemagick.resources.requests.memory="500Mi" ^
+--set alfresco-content-services.libreoffice.resources.requests.memory="500Mi" ^
+--set alfresco-content-services.share.resources.requests.memory="1000Mi" ^
+--set alfresco-content-services.postgresql.resources.requests.memory="500Mi" ^
+--set alfresco-process-services.processEngine.environment.IDENTITY_SERVICE_AUTH="http://localhost-k8s/auth" ^
+--set alfresco-process-services.processEngine.resources.requests.memory="1000Mi" ^
+--set alfresco-process-services.adminApp.resources.requests.memory="250Mi" ^
+--namespace <yourNameSpace>
+```
+
+Repeatedly  run the following command until you can see that all the pods are successfully installed. This can take up to one hour. 
+
+```bash
+kubectl get pods --namespace ayman
+```
+
+If any pods are failing, you can use each of the following commands to see more about their errors:
+
+```bash
+kubectl logs <the part of the pod name that all the pods have in common> --namespace <your namespace name>
+kubectl describe pod <the part of the pod name that all the pods have in common> --namespace <your namespace name>
+```
+
+If the postgres pod won't start and you are getting the following error:
+
+```bash
+2019-02-08 10:27:04.358 UTC [1] FATAL:  data directory "/var/lib/postgresql/data/pgdata" has wrong ownership
+2019-02-08 10:27:04.358 UTC [1] HINT:  The server must be started by the user that owns the data directory.
+```
+
+Run the following commands to delete the storageClass hostpath and set up the hostpath provisioner: 
+
+```bash
+kubectl delete storageclass hostpath
+kubectl create -f https://raw.githubusercontent.com/MaZderMind/hostpath-provisioner/master/manifests/rbac.yaml
+kubectl create -f https://raw.githubusercontent.com/MaZderMind/hostpath-provisioner/master/manifests/deployment.yaml
+kubectl create -f https://raw.githubusercontent.com/MaZderMind/hostpath-provisioner/master/manifests/storageclass.yaml
+```
+
+Clone the following file to your desired location. In the following example, it is cloned to the desktop. To do so, open your command line for git (we used bash), and run the following commands remembering to replace <yourUserName>.
+
+```bash
+cd "C:\Users\<yourUserName>\Desktop"
+git clone https://git.alfresco.com/platform-services/bamboo-build-dbp.git
+```
```

Comment:
I don't think the user who only deploys helm charts will need to clone this repository.

Reply:
So should I delete the instructions related to getting a license?

Reply:
Yes. Customers will obtain licenses from Alfresco and not use the test one stored in `bamboo-build-dbp` (which itself is an Alfresco internal repository).